_episodes/format.md

```diff
@@ -3,51 +3,260 @@ title: "File format"
+- Geospatial data: [NetCDF](https://www.unidata.ucar.edu/software/netcdf/docs/netcdf_data_model.html)
+- Tabular data: [SQLite](https://www.sqlite.org/fileformat.html)
+- Textual data: [Markdown](https://daringfireball.net/projects/markdown/)
+- Image data: [TIFF 6.0 uncompressed](https://www.adobe.io/content/dam/udp/en/open/standards/tiff/TIFF6.pdf)
+- Audio data: [Lossless Audio Codec (FLAC)](https://xiph.org/flac/)
+- Video data: [MPEG-4](https://en.wikipedia.org/wiki/MPEG-4)
+- Documentation: [Microsoft Word](https://en.wikipedia.org/wiki/Microsoft_Word) or [PDF](https://www.iso.org/standard/63534.html)
+- Papers & Articles: [LaTeX](https://www.latex-project.org/)
+- Storyline Definitions: [JSON](https://www.json.org/json-en.html)
 
-non/proprietary format
+> ## Other data types
+>
+> Have a look at the list of the recommended formats for different types (mentioned above)
+> and pick a data type.
+>
+> - What are the other data formats for the type that you selected?
+> - Which of those formats are common in your community?
+{: .discussion}
 
-Recommendation on how to choose a format
+> ## Data Management Plan
+>
+> Data Management Plan (DMP) covers how data can be stored, described and reused.
+{: .callout}
 
-Choose formats common to your field/community
 
-How long do you intend to preserve your data
+## Different formats for different purposes
 
-Use a common, multi-platform file format
+In the previous sections, we learned that different file formats have different properties.
+The purpose of a file should help determine which file format to choose.
+It is good to plan in the beginning of your project, what file formats to use for each purpose:
 
-Check Data repository requirement
+- data collection / processing / analysis,
+- reuse: the longer you want to use the data, the more you have to use open, standard and
+well-documented file formats to avoid obsolescence.
+- preservation: many journals, archives and data repositories require that data are uploaded in certain file formats.
+Therefore, you may have to keep some data files in multiple formats.
 
-More ?
+> ## Non/proprietary format
+>
+> File formats may be either proprietary or non-proprietary(open or free):
+>
+>- The proprietary format is owned by a company, or organization or individual.
+>  Their specifications are usually not publicly available and the risk of obsolescence is high.
+>  If you want to store data in a proprietary format for a reasonable time, consider including a readme.txt file
+>  that documents the name and version of the software used to generate the file, as well as
+>  the information of the company that made the software.
+>
+>- The open format is a file format that is published and free to be used by everybody.
+{: .callout}
 
-Research data management in The Turing Way
+> ## Recommendation about data format
+>
+> Recently, researchers evaluated the utility of dynamical downscaling in agricultural impacts projections,
```

Comment:
Maybe add a link back to the introduction episode (#14), otherwise we're introducing this dataset again and again in every lesson.

Reply:
It is added.